onthequad/onthequad/UserEndPoints.py

```diff
@@ -0,0 +1,89 @@
+import falcon
+import json
+from .UsersDB import UsersDB
+from .DataBaseSetUp import DataBaseSetUp
+authen = DataBaseSetUp.authentication()
+
+class UserEndPoints:
+
+    def on_get(self, req, resp):
+        
+        send = UsersDB.getAllUsers()
+        resp.body = json.dumps(send)
+        resp.status = falcon.HTTP_200
+
+    def on_post(self, req, resp):
+        data = json.loads(req.stream.read())
+    
+        if any(data):
+
+            firstName = data['firstName']
+            lastName = data['lastName']
+            email = data['email']
+            password = data['password']
+
+            user = authen.create_user_with_email_and_password(email, password)
+            uid = user['localId']
```

Comment:
I've been looking into firebase auth and people dont nest the uid like this, they use it as their main ID for the user.

Reply:
deleted

Reply:
https://www.hackanons.com/2018/05/python-django-with-google-firebase.html
Use this as reference to get the uid as the main id. because currently you have one randomly generated when you push.

Reply:
That is where I came up with the uid = user['localId'] and I just deleted it a second ago according to your feedback LOL. Check it out on Develop branch when I have it in Django.

Reply:
You're correct in the localId is the uid, but if you look at firebase the ID does not match the uid for the user.